### PR TITLE
Refactored Reports Component

### PR DESCRIPTION
**Pull request by b7rch4**
Co-authored-by: FractalFraction cln.gardiner@gmail.com

Problem: Our initial solution for displaying the cohort report data contained a very long component

Solution: We have refactored this component, so that it renders a series of smaller components for each section of the report

Jira Issue: https://dfacademy.atlassian.net/jira/software/projects/AT/boards/4?selectedIssue=AT-43

### CODE REVIEW

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -25,47 +26,20 @@ class Reports extends Component {
...
-                    <div id="challengeData">{this.state.reports.challenges.map((challenge) => 
-                                <div>
-                                    <h3 id={"challenge" + challenge.type}>{challenge.type}: </h3>
-                                    <p id={challenge.type + "Percentage"}>Percentage: {challenge.percentage}%</p>
-                                </div>
-                            )
-                        }   
-                    </div>
-                </div>
+                <h2>Gender Distribution</h2>
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports.js:34]: Why don't you pass these headings in as props to the components? It looks like the data is related to the report so feels like it should belong to the component.

File: src/components/Reports/Reports.js [hunk 2]
```diff
@@ -25,47 +26,20 @@ class Reports extends Component {
         return(
             <div>
                 <div id="reportTitle">
```
Comment (dearshrewdwit) [inline src/components/Reports/Reports.js:28]: Looks like this could be a component - pass it cohort data re: name, size and id

**Review by dearshrewdwit — APPROVED** [review]
Nice job breaking this down and finding the category abstraction.

There might be a couple more opportunities here if you're interested to further encapsulate/extract component

**Review by dearshrewdwit — APPROVED** [review]
Nice

**Review by Reginmaru — APPROVED** [review]

**Pull request merged**